fiboa_cli/datasets/nl_crop.py

```diff
@@ -0,0 +1,96 @@
+from ..convert_utils import convert as convert_
+import pandas as pd
+
+
+URI = "https://service.pdok.nl/rvo/brpgewaspercelen/atom/v1_0/downloads/brpgewaspercelen_concept_2023.gpkg"
+
+ID = "nl_crop"
+TITLE = "BRP Crop Field Boundaries for The Netherlands (CAP-based)"
+DESCRIPTION = """
+"BasisRegistratiePercelen" (BRP) combines the location of
+agricultural plots with the crop grown. The data set
+is published by RVO (Netherlands Enterprise Agency). The boundaries of the agricultural plots
+are based within the reference parcels (formerly known as AAN). A user an agricultural plot
+annually has to register his crop fields with crops (for the Common Agricultural Policy scheme).
+A dataset is generated for each year with reference date May 15.
+A view service and a download service are available for the most recent BRP crop plots.
+
+https://service.pdok.nl/rvo/brpgewaspercelen/atom/v1_0/index.xml
+
+Data is currently available for the years 2009 to 2023.
+"""
+BBOX = [2.35417303, 50.71447164, 7.5553525, 55.66948102]
+
+PROVIDER_NAME = "RVO / PDOK"
+PROVIDER_URL = "https://www.pdok.nl/introductie/-/article/basisregistratie-gewaspercelen-brp-"
+ATTRIBUTION = None
+LICENSE = "CC0"  # http://creativecommons.org/publicdomain/mark/1.0/
```

Comment:
see above, this needs to be clarified

Reply:
Also for crops, the data is published with [WFS](https://service.pdok.nl/rvo/brpgewaspercelen/wfs/v1_0?request=GetCapabilities&service=WFS) under CC0. I could document this.

Reply:
Yeah, as above I agree that we can use CC0-1.0 with a comment about why we chose it as above.